src/client/components/OrganizationPanel.js

```diff
@@ -1,24 +1,64 @@
-import React from 'react'
-import styled from 'styled-components'
-import { Text } from '@aragon/ui'
+import React, { useContext } from 'react'
 import CardWrapper from '../wrappers/styleWrappers/CardWrapper'
-
-const addMore = () => console.log('add more orgs')
+import { useProfile } from '../hooks'
+import { ModalContext } from '../wrappers/modal'
+import EducationHistoryTile from './EducationHistoryTile'
+import { open, removeItem } from '../stateManagers/modal'
+import { Text, theme } from '@aragon/ui'
+import styled from 'styled-components'
 
 const OrganizationPanel = () => {
+  const { organizations, viewMode } = useProfile()
+  const { dispatchModal } = useContext(ModalContext)
+
+  const organizationsNotEmpty = Object.keys(organizations).length > 0
+
+  const cardProps = {
```

Comment:
I think this is _okay_ for now, but it would be nice to remove any unnecessary variables if possible. If we could just add these as props that would save us another variable here. But i think this pattern exists elsewhere in the app, so it might be easier to just make one sweeping change. What are your thoughts?

Reply:
if only "Object.keys(organizations).length" could be expressed in more concise way... How about replacing it with _.isEmpty() from lodash? Remove extra variable, replace the length test with isEmpty?

Reply:
I was referring to `cardProps`, not `organizationNotEmpty`. But i don't think we should add another dependency just to make that check. The check you're making is fine and can still be written in one line anyways.